File: bitget-golang-sdk-api/pkg/model/spot/order/batchordersreq.go
package order

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: spot batch order request
 */
type BatchOrdersReq struct {

	/**
	 * order list
	 */
	OrderList []SpotOrdersReq `json:"orderList"`
	/**
	 * Currency pair
	 */
	Symbol string `json:"symbol"`
}
type SpotOrdersReq struct {
	/**
	 * Order direction
	 */
	Side string `json:"side"`
	/**
	 * Order type
	 */
	OrderType string `json:"orderType"`
	/**
	 * Order Control Type
	 */
	Force string `json:"force"`
	/**
	 * Entrusted price, only applicable to price limit order
	 */
	Price string `json:"price"`
	/**
	 * quantity
	 */
	Quantity string `json:"quantity"`
	/**
	 * Client order ID
	 */
	ClientOrderId string `json:"clientOrderId"`
}

File: bitget-node-sdk-api/src/lib/model/spot/order/BatchOrdersReq.ts
import {OrdersReq} from './OrdersReq';

export class BatchOrdersReq {
    private _symbol!:string;
    private _orderList!:OrdersReq[];


    get symbol(): string {
        return this._symbol;
    }

    set symbol(value: string) {
        this._symbol = value;
    }

    get orderList(): OrdersReq[] {
        return this._orderList;
    }

    set orderList(value: OrdersReq[]) {
        this._orderList = value;
    }
}